Database/Updates/World/04-06-20-2017-alterPostionTable-stored.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `ace_world`.`ace_position`
+  DROP COLUMN `landblock` ,
+  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) STORED,
```

Comment:
@Mogwai-TheFurry The impact was noticed on the first in-game load in of each place I visited, for example I loaded the server, logged in Arwic, exit portal, see empty town for a few seconds then objects populated, teleport to Holtburg, see same behavior with a variance of the length of time the load in takes, eventually return to Arwic and its town is loaded instantly back to the client which would make the think there is some caching in memory going on?